README.md

```diff
@@ -169,6 +173,13 @@ To reference the OpenIddict MyGet feed, **create a `NuGet.config` file** (at the
     <img src="https://volosoft.com/assets/logos/volosoft-logo-dark.svg" width="500px" alt="Volosoft logo" />
   </picture>
 </a>
+<p></p>
+<a href="https://openiddictcomponents.com">
```

Comment:
Looks like `https://openiddictcomponents.com/` redirects to `https://www.openiddictcomponents.com/` so let's avoid the unnecessary redirection 😃 

```suggestion
<a href="https://www.openiddictcomponents.com/">
```